lib/command/info.rb

```diff
@@ -184,8 +170,7 @@ def print_missing_apps_starting_with
            "(replace 'whatever' with whatever suffix you want):"
 
       @missing_apps_starting_with.each do |app, _workloads|
-        app_with_suffix = "#{app}#{app.end_with?('-') ? '' : '-'}whatever"
-        puts "  - `cpl setup-app -a #{app_with_suffix}`"
+        puts "  - `cpl setup-app -a #{app}-whatever`"
```

Comment:
We don't need to do this check, we can just add the suffix.

Reply:
What is the "whatever" I think we can give a better message.

Reply:
```
puts " - `cpl setup-app -a #{app}-pr-1234`"
```

Reply:
OK -- I see you defined whatever above.